package.json

```diff
@@ -3,23 +3,30 @@
 	"version": "8.0.1",
 	"description": "`pngquant` wrapper that makes it seamlessly available as a local dependency",
 	"license": "GPL-3.0+",
-	"repository": "imagemin/pngquant-bin",
+	"homepage": "https://github.com/imagemin/pngquant-bin#readme",
```

Comment:
These are intended, please revert lines 6-13

Reply:
You're right, I didn't know about this shortcut notation (ran `npm view pngquant-bin bugs`). No point in getting this merged then